openwisp_monitoring/monitoring/charts.py

```diff
@@ -161,7 +161,10 @@
         'order': 260,
         'query': {
             'influxdb': (
-                "SELECT 100 * MEAN(load_1) AS CPU_load FROM {key} WHERE "
+                "SELECT 100 * MEAN(load_1) / MEAN(cpus) AS Average_1m, "
```

Comment:
let's avoid performing this division each time, we don't need to store the number of CPUs in the timeseries, because as you noted @nepython, this never changes.
I'm not sure if @PabloCastellano told you to do this, but I advise against doing this because it has no value, it only adds operational costs.

Reply:
@nemesisdesign Yes, that was my idea because IMO influxdb should always have ther raw values but if you prefer to do it otherwise I'm fine with that